frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -183,14 +187,14 @@ export const getSections = createSelector(
     }
 );
 
-export const getActiveSectionName = (state) => state.router && state.router.location && state.router.location.query.section
+export const getActiveSectionName = (state, props) => props.params.section
 
 export const getActiveSection = createSelector(
     getActiveSectionName,
     getSections,
-    (section = "Setup", sections) => {
+    (section = "setup", sections) => {
         if (sections) {
-            return _.findWhere(sections, {name: section});
+            return _.findWhere(sections, { slug: section });
```

Comment:
I usually prefer to build a map keyed on `slug` and use that whenever I find myself having to use find like this, but probably not be worth the risk and effort to refactor something that already works in this particular case.